clients/gtest/blas1_gtest.cpp

```diff
@@ -2286,6 +2300,24 @@ TEST_P(blas1_gtest, amax_batched_float_complex)
     }
 }
 
+TEST_P(blas1_gtest, amax_batched_double_complex)
+{
+    Arguments       arg    = setup_blas1_arguments(GetParam());
+    hipblasStatus_t status = testing_amax_batched<hipblasDoubleComplex>(arg);
+
+    if(status != HIPBLAS_STATUS_SUCCESS)
+    {
+        if(arg.batch_count < 0)
+        {
+            EXPECT_EQ(HIPBLAS_STATUS_INVALID_VALUE, status);
+        }
+        else
+        {
+            EXPECT_EQ(HIPBLAS_STATUS_SUCCESS, status); // fail
```

Comment:
All the // fail comments not really sure of.

Reply:
Yeah, that line just ensures that we actually get a gtest failure when we don't get HIPBLAS_STATUS_SUCCESS from testing_*.hpp. `// fail` is meant to imply that if we get to that line, the test will fail.